test/e2e/cli/version_test.go

```diff
@@ -67,7 +67,7 @@ var _ = Describe("kraft version", func() {
 
 			Expect(stderr.String()).To(BeEmpty())
 			Expect(stdout.String()).To(MatchRegexp(
-				`^(level=error msg="unknown command "some-arg" for "kraft version"")|(unknown command "some-arg" for "kraft version")\n$`,
+				`^{"level":"error","msg":"unknown command \\"some-arg\\" for \\"kraft version\\""}\n$`,
```

Comment:
nit: We could as well use `Equal()` here if we are gating the string between `^$`.